### PR TITLE
bug: error message for alphabets and multiple decimals

### LINKED ISSUE #159: [C4GT Community]: Add instructions / error when wrong characters are entered in numerical textbox
## Description
Say a second decimal point is entered - currently we are just preventing the character in textbox, maybe we can display an error saying 'you've already entered decimal point' . Similar error when an alphabet is entered.

## Goals
- [ ] Instructions when bad characters are entered in numerical textbos

### Expected Outcome

Instructions when bad characters are entered in numerical textbos

### Acceptance Criteria

_No response_

### Implementation Details

See Body.vue

### Mockups/Wireframes

_No response_

### Product Name

Quiz Frontend

### Organisation Name

Avanti Fellows

### Domain

Education

### Tech Skills Needed

Vue.js

### Mentor(s)

@suryabulusu 

### Complexity

Low

### Category

Frontend

### PR DESCRIPTION
**Pull request by kesharibhai84**
Fixes #159 

## Summary
 Error message for alphabets and multiple decimals added.

### Error message for _Multiple Decimal_
image

### Error message for _Alphabets_
image

- [X] Test Responsiveness
  - [x] Laptop (1200px)
  - [x] Tablet (760px)
  - [X] Phone (320px)
- [X] Cross-Browser Testing
  - [X] Chrome
  - [X] Firefox
- [X] ~Local Language Support~
- [X] Hygiene and Housekeeping
  - [X] Self-review
  - [X] Comments have been added appropriately
  - [X] Check for bundle size [here](https://bundlephobia.com/) if adding a package
  - [X] Added relevant details like Labels/Projects/Milestones etc.
  - [X] If adding or removing any environment variable, update `docs/ENV.md`, `.env.example` and the Github workflows.
- [X] Testing
  - [X] Wrote tests
  - [X] Tested locally
  - [X] Tested on staging
  - [X] Tested on an actual physical phone
  - [X] Tested on production
- [X] Lighthouse Checks
  - [X] Images have `alt` attributes
  - [X] Any `` tags have `width` and `height` specified
  - [X] Any `target="_blank"` links have `rel="noopener"`
  - [X] Only SVGs are used as images. If PNGs are used, their size has been optimised.
  - [X] Any SVG buttons without text have their `aria-label` attributes set

### CODE REVIEW

File: src/components/Omr/OmrItem.vue [hunk 1]
```diff
@@ -386,279 +442,363 @@ export default defineComponent({
...
+            (doesNumberContainDecimal(state.numericalAnswer) ||
+              state.numericalAnswer == null))
         ) {
-          event.preventDefault()
+          if (
+            event.data == "." &&
+            doesNumberContainDecimal(state.numericalAnswer)
+          ) {
+            console.error("Decimal point is already present !");
+            showErrorNotification("You've already entered decimal point !");
```
Comment (suryabulusu) [inline src/components/Omr/OmrItem.vue:558]: "You have already entered a decimal point!"

File: src/components/Omr/OmrItem.vue [hunk 2]
```diff
@@ -386,279 +442,363 @@ export default defineComponent({
...
         // in other cases, return to escape null type error
-        return
+        return;
       }
+
+      const isAlphabet = /[a-zA-Z]/.test(event.data); // Check if the input is an alphabet
+
+      if (isAlphabet) {
+        console.error("Alphabets are not allowed !");
+        showErrorNotification("Alphabets are not allowed !");
```
Comment (suryabulusu) [inline src/components/Omr/OmrItem.vue:517]: Alphabets are not allowed!

**Review by suryabulusu — APPROVED** [review]
approved!

**Comment by VedantKhairnar** [comment]
Hello @kesharibhai84   ,
Thanks for your contribution.
As part of Augtoberfest, we want to reward you for your contributions, so pls fill out [this form](https://forms.gle/VBDUPWMYcZEbmVrQ9) by 19th Sept.
Thanks.

**Pull request merged**